README.md

```diff
@@ -246,27 +246,9 @@ This application provides container images for demonstration purposes. The base
 
 ## Notice for Docker image
 
-DockerHub: [https://hub.docker.com/r/tractusx/sdfactory](https://hub.docker.com/r/tractusx/sdfactory)  <br />
-Eclipse Tractus-X product(s) installed within the image:
+Docker Notice for this application can be find below:
 
-__SD Factory__
-
-- GitHub: https://github.com/eclipse-tractusx/sd-factory
-- Project home: https://projects.eclipse.org/projects/automotive.tractusx
-- Dockerfile: https://github.com/eclipse-tractusx/sd-factory/blob/main/Dockerfile
-- Project license: [Apache License, Version 2.0](https://github.com/eclipse-tractusx/sd-factory/blob/main/LICENSE)
-
-**Used base image**
-
-- [eclipse-temurin:17.0.8.1_1-jdk](https://github.com/adoptium/containers)
-- Official Eclipse Temurin DockerHub page: https://hub.docker.com/_/eclipse-temurin
-- Eclipse Temurin Project: https://projects.eclipse.org/projects/adoptium.temurin
-- Additional information about the Eclipse Temurin images: https://github.com/docker-library/repo-info/tree/master/repos/eclipse-temurin
-
-As with all Docker images, these likely also contain other software which may be under other licenses 
-(such as Bash, etc. from the base distribution, along with any direct or indirect dependencies of the primary software being contained).
-
-As for any pre-built image usage, it is the image user's responsibility to ensure that any use of this image complies with any relevant licenses for all software contained within.
+ [DOCKER_NOTICE.md](DOCKER_NOTICE.md)
```

Comment:
I'm not certain about removing the section and referencing the dedicated notice @SebastianBezold what you think?

Reply:
Hi @evegufy and @adkumar1,

in my opinion it is fine as it is. The content should only be defined once in a dedicated `.md`. The top-level `README.md` should still link to it, like it currently does. 
Just not sure if this is a change, that is worth mentioning in the CHANGELOG

Reply:
Removed it from CHANGELOG then.

Reply:
ok, that's fine.
Regarding changelog: I think it's worth mentioning but as there was already a docker notice, I'd add it under changed like this: "Provided docker notice in dedicated file"